gcc/config/mips/ps2sdk.h

```diff
@@ -4,6 +4,8 @@
     --start-group \
         %{g:-lg} %{!g:-lc} \
         -lcdvd \
+        -lpthread \
+        -lpthreadglue \
```

Comment:
Is there a reason these libraries are inside the group? Is there a circular dependency in these 2 new libraries?

Reply:
Both pthread and pthreadglue use newlib functions, additionally pthreadglue uses functions from kernel

Reply:
Is the never ending story